### ISSUE TITLE
Docs: update Hardware & system requirements (for Integration > Deploy to Prod)

### ISSUE THREAD

**Issue by tselit** [interledger/rafiki#2933]
## Context
Add content to this section: Integration > Deployment > Deploy to production > Hardware & system requirements

## Todos
- [ ] setup meetings with Max the minimum hardware and system requirements for a Rafiki Production deployment
- [ ] document the the minimum hardware and system requirements for a Rafiki Production deployment

**Comment by mkurapov** [comment]
@hajjimo can we close this issue now that we will have the information in the deployment page?

**Comment by hajjimo** [comment]
Yes, closing as we'll document system requirements on the deployment guide.
